sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Primitives/MessageProcessor.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.Azure.WebJobs.ServiceBus
     /// <summary>
     /// This class defines a strategy used for processing Service Bus messages.
     /// </summary>
-    public class MessageProcessor
+    internal class MessageProcessor
```

Comment:
Does this constitute a breaking change?   _(and does that matter in this context?)_

Reply:
Reverted this change.